.github/workflows/ci.yml

```diff
@@ -53,9 +53,15 @@ jobs:
         shell: bash
         run: echo "INSTA_SNAPSHOT_REFERENCES_FILE=$(mktemp)" >> "${GITHUB_ENV}"
 
-      - name: Run the test suite
+      - name: Run the test suite (Without AWS KMS)
+        if: ${{ runner.os != 'Linux' }}
         shell: bash
-        run: cargo test --timings --workspace --locked
+        run: cargo test --timings --workspace --locked -- --test-threads=1
+
+      - name: Run the test suite (With AWS KMS)
+        if: ${{ runner.os == 'Linux' }}
+        shell: bash
+        run: cargo test --timings --workspace --locked --features aws-kms -- --test-threads=1
```

Comment:
I have added `-- test-threads=1` to both of these.  I wonder if I can avoid this for the top one.